mirgecom/multiphysics/thermally_coupled_fluid_wall.py

```diff
@@ -783,112 +894,149 @@ def get_interface_boundaries(
 
         An identifier denoting a particular quadrature discretization to use during
         operator evaluations.
+
+    comm_tag: Hashable
+        Tag for distributed communication
     """
-    if interface_noslip:
-        fluid_bc_class = InterfaceFluidNoslipBoundary
-    else:
-        fluid_bc_class = InterfaceFluidSlipBoundary
+    fluid_interface_boundaries_no_grad, wall_interface_boundaries_no_grad = \
+        _get_interface_boundaries_no_grad(
```

Comment:
Is it necessary that this tiny function calls a bigger function? Why not have one function only?

Reply:
I rather like this encapsulation for the interface getters.

Reply:
I don't have a strong opinion about this. I suspect at one point the `_get_*` functions were called in multiple places, but now that they're only called from the corresponding `add_*` it looks like they could be safely removed. You guys decide what I should do and I'll do it. 🙂

Reply:
We decided that you can leave it as it is. Moreover, we decided that you have to buy as a whisky.

Reply:
> we decided that you have to buy us a whisky.
+10 👍 

I also don't feel strongly about this.  If it is safe to remove, and @tulioricci strongly dislikes then it should be inlined.